tests/integration/targets/win_route/tasks/main.yml

```
@@ -0,0 +1,34 @@
+---
+# test code for the win_psmodule module when using winrm connection
+# (c) 2017, Daniele Lazzari <lazzari@mailup.com>
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+- block:
+  - name: get os info
+    ansible.windows.win_shell: '[Environment]::OSVersion.Version -ge [Version]"6.3"'
+    register: os
+
+  - name: run all tasks
+    include_tasks: tests.yml
+    when: os.stdout_lines[0] == "True"
```

The baseline version is Server 2016 so we don't need this check anymore.

```suggestion
  - name: run all tasks
    include_tasks: tests.yml
```